examples/pcc-dialin-server/simple-python/server.py

```diff
@@ -0,0 +1,197 @@
+#
+# Copyright (c) 2025, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+# server.py
+
+
+import base64  # for calculating hmac signature
+import hmac
+import os  # for accessing environment variables
+import time  # for setting expiration time
+from typing import Any, Dict, List, Optional
+
+import requests
+from dotenv import load_dotenv
+from fastapi import FastAPI, HTTPException, Request
+from loguru import logger
+from pydantic import BaseModel, Field
+
+load_dotenv(override=True)
+
+app = FastAPI()
+
+
+class RoomRequest(BaseModel):
+    test: Optional[str] = Field(None, alias="Test", description="Test field")
```

Comment:
Does this do anything? Can we remove it?

Reply:
It is the first request sent by Daily when the `pinless_dialin` is setup, i.e., the webhook receives a body with `test`